.gitignore

```diff
@@ -1,170 +1,7 @@
-# Byte-compiled / optimized / DLL files
+# ignore python cache directory
 __pycache__/
-*.py[cod]
-*$py.class
 
-# C extensions
-*.so
+# ignore poetry lock file - created automatically
+poetry.lock
```

Comment:
[Poetry recommends committing the lockfile](https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control) to get more reproducible builds. It makes it so that when dependencies are installed, the exact ones are used for different developers.